### ISSUE TITLE
FSharp.Charting 0.90.10 references  gtk-sharp-3

### ISSUE THREAD

**Issue by juergenhoetzel** [fslaborg/FSharp.Charting#76]
Although the OxyPlot.GtkSharp assembly is still using gtk-sharp-2:
```
juergen@samson:~/shared/fsharp/GtkTest → monodis ./src/GtkTest/bin/Debug/OxyPlot.GtkSharp.dll --assemblyref|head -n 30
AssemblyRef Table
...
3: Version=2.12.0.0
	Name=gtk-sharp
	Flags=0x00000000
	Public Key:
```

GTK-3 and GTK-2 is not API/ABI compatible. A simple test program results in :
```

(GtkTest:24195): GLib-GObject-CRITICAL **: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed

(GtkTest:24195): GLib-GObject-CRITICAL **: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed

(GtkTest:24195): GLib-GObject-CRITICAL **: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed

(GtkTest:24195): GLib-GObject-CRITICAL **: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed

(GtkTest:24195): GLib-GObject-CRITICAL **: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed

(GtkTest:24195): GLib-GObject-CRITICAL **: g_boxed_type_register_static: assertion 'g_type_from_name (name) == 0' failed
```

0.90.9  works find works fine on my GNU/Linux system.

Seems to be caused by this commit 9ecbe70d2ea7dfec08336978141c8679b6bfa5cd

**Comment by dsyme** [comment]
Definitely needs to be fixed! :)

**Comment by dsyme** [comment]
It looks like the GtkSharp nuget package is GTK# 3.0.  We really need a package for GTK# 2.12. Any chance someone can post one?

**Comment by juergenhoetzel** [comment]
@dsyme BTW: Could you explain the purpose of the conditional compilation in the Gtk implementation:

    #if INCOMPLETE_API
    ... member public x.SaveChartAs(filename : string, format : ChartImageFormat) 
    #endif

I wonder why these members are missing by default.

Thanks!

**Comment by dsyme** [comment]
I think they just haven't been implemented. By all means we should complete theme.

**Comment by dsyme** [comment]
Fixed by #79
